elfpy/utils/logs.py

```diff
@@ -19,39 +19,39 @@ def setup_logging(
     max_bytes: int | None = None,
     log_level: int | None = None,
     delete_previous_logs: bool = False,
-    log_stdout: bool = True,
```

Comment:
leave this true if possible to avoid accidental problems downstream

Reply:
or at a minimum double check the elfpy/examples/eth_bots endpoint to make sure we aren't changing the behavior from default logging to stdout

Reply:
left it as True